frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -2336,4 +2413,8 @@ function ReaderFooter:onScreenResize()
     self:resetLayout(true)
 end
 
+function ReaderFooter:onTimeFormatChanged()
+    self:refreshFooter(true, true)
+end
+
```

Comment:
@poire-z : Can there be a problem with that?
`maybeUpdateFooter` is not applicable here, as the `12-hour-clock` can be selected from menu.

Reply:
Not really familiar with refreshFooter() and maybeUpdateFooter , but @NiLuJe is.
(But I'd say it's really edge casey: one is toggling the format in this remove menu: if it doesn't see it updated immediately, I wouldn't mind: he will see it on the next page change.)

Reply:
;) I would get upset (a bit), if I toggle a switch and the footer does not react.

But you are totally right: I would expect a user only toggling that once to his/her preferences and never touch this again. So overwriting something different would only occur once.

Reply:
I meant that Time format change is far _remote_ in the `Device>` submenu, so it having no immediate effect on the footer is understandable/bearable.
If it were in the `Status bar>` submenu, then it might be more of an issue as all the other settings have an immediate effect.